From 0ae60c9c601b6f62fab0ad8f67f26683c6710071 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 8 Jan 2020 16:42:01 -0800
Subject: [PATCH] Block LTO when detecting functions

---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index ffb5db7..5f34bab 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -170,7 +170,7 @@ ESQ
 
   close(TST);
   print "Looking for $f()" . "." x (13-length($f)) . " ";
-  if (system("$cfg{'cc'} $flags $funcs{$f} functest_$f.c > functest_$f.log 2>&1")) {
+  if (system("$cfg{'cc'} $flags -fno-lto $funcs{$f} functest_$f.c > functest_$f.log 2>&1")) {
     print "not found.\n";
   } else {
     $define{"-DHAVE_\U$f"} = undef;
-- 
2.24.1

